### PR TITLE
Update README for MediaTek backend

### PR DESCRIPTION
**Pull request by neuropilot-captain**
1. Move common setup instructions under backends/
2. Refactoring
3. Add link for NeuroPilot Express SDK portal

### CODE REVIEW

**Comment by neuropilot-captain** [comment]
Hi @cccclai,

We have added the portal link to our README. Thanks!

**Review by cccclai — APPROVED** [review]
Looks good, thank you!

**Review by cccclai — APPROVED** [review]

File: backends/mediatek/scripts/README.md [hunk 1]
```diff
@@ -10,41 +10,60 @@ Before you begin, ensure you have the following prerequisites installed and conf
...
-export ANDROID_NDK=<path_to_android_ndk>
-```
+   ```bash
+   export ANDROID_NDK=<path_to_android_ndk>
+   ```
 
 ### 3. MediaTek ExercuTorch Libraries
 
-Download the following libraries from MediaTek's NeuroPilot portal (link to be added):
+Download [NeuroPilot Express SDK](https://neuropilot.mediatek.com/resources/public/npexpress/en/docs/npexpress) from MediaTek's NeuroPilot portal:
```
Comment (cccclai) [inline backends/mediatek/scripts/README.md:27]: It reads a bit weird, "MediaTek's NeuroPilot portal" sounds like the link...

**Pull request closed without merging**